install_config/install/advanced_install.adoc

```diff
@@ -1678,22 +1675,113 @@ specifications, and save it as *_/etc/ansible/hosts_*.
+and configuration files available on the local host. To run the installer, use
+the following command, specifying `-i` if your inventory file located somewhere
+other than *_/etc/ansible/hosts_*:
 
 ----
 ifdef::openshift-enterprise[]
-# ansible-playbook /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml
+# ansible-playbook  [-i /path/to/inventory] \
+    /usr/share/ansible/openshift-ansible/playbooks/byo/config.yml
 endif::[]
 ifdef::openshift-origin[]
-# ansible-playbook ~/openshift-ansible/playbooks/byo/config.yml
+# ansible-playbook [-i /path/to/inventory] \
+    ~/openshift-ansible/playbooks/byo/config.yml
 endif::[]
 ----
 
 If for any reason the installation fails, before re-running the installer, see
 xref:installer-known-issues[Known Issues] to check for any specific
 instructions or workarounds.
 
+[[running-the-advanced-installation-system-container]]
+=== Running the Containerized Installer
+
+include::install_config/install/advanced_install.adoc[tag=syscontainers_techpreview]
+
+The 
+ifdef::openshift-enterprise[]
+*openshift3/ose-ansible*
+endif::[]
+ifdef::openshift-origin[]
+*openshift/origin-ansible*
+endif::[]
+image is a containerized version of the {product-title} installer that runs as a
+link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux_atomic_host/7/html/managing_containers/running_system_containers[system container]. System containers are stored and run outside of the traditional
+*docker* service. Functionally, using the containerized installer is the same as
+using the traditional RPM-based installer, except it is running in a
+containerized environment instead of directly on the host.
+
+. Use the Docker CLI to pull the image locally:
++
+----
+ifdef::openshift-enterprise[]
+$ docker pull registry.access.redhat.com/openshift3/ose-ansible:v3.6
+endif::[]
+ifdef::openshift-origin[]
+$ docker pull docker.io/openshift/origin-ansible:v3.6
+endif::[]
+----
+
+. The installer system container must be stored in
+link:https://access.redhat.com/documentation/en-us/red_hat_satellite/6.2/html/content_management_guide/managing_ostree_content[OSTree]
+instead of defaulting to *docker* daemon storage. Use the Atomic CLI to import
+the installer image from the local *docker* engine to OSTree storage:
++
+----
+$ atomic pull --storage ostree \
+ifdef::openshift-enterprise[]
+    docker:registry.access.redhat.com/openshift3/ose-ansible:v3.6
+endif::[]
+ifdef::openshift-origin[]
+    docker:docker.io/openshift/origin-ansible:v3.6
+endif::[]
+----
+
+. Install the system container so it is set up as a systemd service:
++
+----
+$ atomic install --system \
+    --storage=ostree \
+    --name=openshift-installer \//<1>
+    --set INVENTORY_FILE=/path/to/inventory \//<2>
+ifdef::openshift-enterprise[]
+    docker:registry.access.redhat.com/openshift3/ose-ansible:v3.6
+endif::[]
+ifdef::openshift-origin[]
+    docker:docker.io/openshift/origin-ansible:v3.6
+endif::[]
```

Comment:
@adellape that will pull the image from the local Docker engine.  You won't need to do that if the image can be pulled from the registry.

What issue are you having if you use directly `docker.io/openshift/origin-ansible:v3.6` ?